server/rucio.conf.j2

```diff
@@ -44,6 +44,10 @@ CacheRoot /tmp
 {% endif %}
  SSLVerifyClient optional_no_ca
  SSLVerifyDepth 10
+ <Location /auth/x509>
```

Comment:
this should probably be only applied on the first virtual-host below

Reply:
Good catch. I've made the necessary changes :)